operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -316,6 +327,23 @@ local utils = import 'mixin-utils/utils.libsonnet';
       fill: 0,
     },
 
+  // The provided containerName should be a regexp from $._config.container_names.
+  containerGoHeapInUsePanel(containerName)::
+    $.panel('Memory (go heap inuse)') +
+    $.queryPanel(
+      'sum by(%(instanceLabel)s) (go_memstats_heap_inuse_bytes{%(namespaceMatcher)s,container=~"%(containerName)s"})' % {
```

Comment:
As expected was broken. Fixed here: https://github.com/grafana/mimir/pull/3497/commits/a3c0f5e4815f92101549dacf7bb122703be59b59. The baremetal queries are a bit weird, because they include helm-compatible name matcher and a double ".*.*" at the end. The queries work, they're just uglier than needed. I will fix it in a follow up PR to keep diff easier to review.

Reply:
> they include helm-compatible name matcher
This is address here: https://github.com/grafana/mimir/pull/3502